docs/source/use_cases/discover_google_data/1000_genomes_ld.rst

```diff
@@ -0,0 +1,25 @@
+1,000 Genomes Phase 3 Linkage Disequilibrium Data
+=================================================
+
+.. comment: begin: goto-read-the-docs
+
+.. container:: visible-only-on-github
+
+   +-----------------------------------------------------------------------------------+
+   | **The properly rendered version of this document can be found at Read The Docs.** |
+   |                                                                                   |
+   | **If you are reading this on github, you should instead click** `here`__.         |
+   +-----------------------------------------------------------------------------------+
+
+.. _RenderedVersion: http://googlegenomics.readthedocs.org/en/latest/use_cases/discover_google_data/1000_genomes_ld.html
```

Comment:
This pull request is into the "staging" branch. The github link explicitly goes to "latest".
Once this PR is merged into latest, then the link will work.